src/index.ts

```diff
@@ -35,3 +35,6 @@ export { mapOutput } from './mapOutput'
 
 // UUID
 export { uuid } from './string/uuid'
+
+// io-ts
+export { fallback } from './io-ts/fallback'
```

Comment:
I think that we can get rid of the `io-ts` folder (for example `mapOutput.ts` is already in the root folder)